app/views/contracts/show.html.erb

```diff
@@ -1 +1,31 @@
-<h1>Hi!</h1>
\ No newline at end of file
+<h1> Hi! </h1>
+<table class="table">
+  <thead class="thead-inverse">
+      <tr> 
+        <th> ID:  </th> 
+        <th> Customer Request ID: </th>
+        <th> Company ID:  </th>
+        <th> Material Cost Estimate:   </th>
+        <th> Labor Cost Estimate  </th>
+        <th> Completion Date:  </th>
+        <th> Notes:  </th>
+        <th> Accepted:  </th>
+      </tr>
+  </thead>
+
+  <tbody>
+      <tr>
+        <td> <%= @contract.id %> </td>
+        <td> <%= @contract.quote.customer_request_id %> </td>
+        <td> <%= @contract.quote.company_id %> </td>
+        <td> <%= @contract.quote.materials_cost_estimate %>  </td>
+        <td> <%= @contract.quote.labor_cost_estimate %> </td>
+        <td> <%= @contract.quote.completion_date_estimate %> </td>
+        <td> <%= @contract.quote.notes %> </td>
+        <td> <%= @contract.quote.accepted %> </td>
+      </tr>
+  </tbody>
+</table>
+<input type="Complete?" class="btn btn-primary" value="Complete?">
```

Comment:
input type tags are for the type of data the input is for like, text, date etc..

take a look here:
https://www.w3schools.com/tags/att_input_type.asp